### PR TITLE
documentation: Updated git checkout command and reorder unit test steps

### PR DESCRIPTION
**Pull request by ca-nguyen**
*Issue #, if available:*

*Description of changes:*
- Updated git checkout command to checkout main (and not master). 
- Reordered unit test steps: pip install must be done inside the project folder.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by wong-a — DISMISSED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ You should only worry about manually running any new integration tests that you
 
 1. Create a new git branch:
      ```shell
-     git checkout -b my-fix-branch master
+     git checkout -b my-fix-branch main
```
Comment (wong-a) [inline CONTRIBUTING.md:83]: `<start_point>` (the second argument after `-b`) is optional. After forking and cloning the repository, your `HEAD` should be at the latest commit in the default branch already.

Comment (ca-nguyen) [reply CONTRIBUTING.md:83]: Agreed! I can add a comment to specify that it is optional if you are starting from main branch.

Comment (wong-a) [reply CONTRIBUTING.md:83]: Or just remove it

Comment (ca-nguyen) [reply CONTRIBUTING.md]: Done!

**Review by wong-a — APPROVED** [review]

**Review by shivlaks — APPROVED** [review]

**Pull request merged**